awscli/customizations/s3/subcommands.py

```diff
@@ -1276,6 +1286,19 @@ def _validate_path_args(self):
             if self._should_emit_validate_s3_paths_warning():
                 self._emit_validate_s3_paths_warning()
 
+        # If we're using the checksum-algorithm flag, the path type must be locals3
```

Comment:
We can remove this comment. It's just describing what the code is doing and doesn't have any explanatory benefits.